jobs/settings.py

```diff
@@ -291,3 +293,12 @@
 }
 
 NOTIFICATIONS_ASYNC_QUEUE_NAME = CELERY_LOW_QUEUE_NAME
+
+# SMTP Setup
+
+EMAIL_BACKEND = env("EMAIL_BACKEND", default="django.core.mail.backends.console.EmailBackend")
+EMAIL_USE_TLS = env("EMAIL_USE_TLS", default=False)
+EMAIL_HOST = env("EMAIL_HOST", default="")
+EMAIL_PORT = env("EMAIL_PORT", default="")
+EMAIL_HOST_USER = env("EMAIL_HOST_USER", default="")
+EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD", default="")
```

Comment:
This one is a secret. Should be without default. This way you are forcing to add a password in the environment vars when we will do the deploy.

Reply:
When you will do that, run `make test` and won't work, you will need to add this secret in the docker-compose.testing.yml like the postgres password and so on.

Reply:
Thanks, I was not aware of it. :+1: